.gitignore

```diff
@@ -9,7 +9,7 @@ CMakeFiles/
 .vs/
 libuv/
 raft_tests.md
-tests/raft_scenarios
+tests/raft_scenarios/scenario-*
```

Comment:
Making this more precise to allow additional scenarios to be added. These `scenario-X` files are now actually generated in the build directory (or local directory, wherever `raft_scenarios_runner` is executed), but I don't want to add the scenarios you all have sitting around